0019/0019-endpoint-metadata-patch-request.avsc

```diff
@@ -0,0 +1,37 @@
+{
+    "namespace":"org.kaaproject.ipc.epmmp.gen.v1",
+    "name":"EndpointMetadataPatchRequest",
+    "type":"record",
+    "doc":"Interservice endpoint parcial metadata update request",
+    "fields":[
+        {
+            "name":"correlationId",
+            "type":"string",
+            "doc":"Message ID primarily used to track message processing across services"
+        },
+        {
+            "name":"timestamp",
+            "type":"long",
+            "doc":"Message creation UNIX timestamp in milliseconds"
+        },
+        {
+            "name":"timeout",
+            "type":"long",
+            "default":0,
+            "doc":"Amount of milliseconds since the timestamp until the message expires. Value of 0 is reserved to indicate no expiration."
+        },
+        {
+            "name":"endpointId",
+            "type": "string",
+            "doc":"Identifier of the endpoint, on behalf of which metadata is requested"
+        },
+        {
+            "name":"operations",
+            "type":{
+                "type":"array",
+                "items":"EndpointMetadataPatchOperations"
+            },
+            "doc":"List of metadata operations"
+        }
+    ]
+}
```

Comment:
Forgot about conditional requests (are used by EPMX): 
```
{
    "namespace": "org.kaaproject.ipc.epmmp.gen.v1",
    "name": "EndpointMetadataPatchRequest",
    "type": "record",
    "doc": "Interservice endpoint partial metadata update request",
    "fields": [
        {
            "name": "correlationId",
            "type": "string",
            "doc": "Message ID primarily used to track message processing across services"
        },
        {
            "name": "timestamp",
            "type": "long",
            "doc": "Message creation UNIX timestamp in milliseconds"
        },
        {
            "name": "timeout",
            "type": "long",
            "default": 0,
            "doc": "Amount of milliseconds since the timestamp until the message expires. Value of 0 is reserved to indicate no expiration."
        },
        {
            "name": "endpointId",
            "type": "string",
            "doc": "Identifier of the endpoint, on behalf of which metadata is requested"
        },
        {
            "name": "ifMatch",
            "type": [
                "null",
                "string"
            ],
            "default": null,
            "doc": "Makes the request conditional"
        },
        {
            "name": "patch",
            "type": "bytes",
            "doc": "Patch payload as defined in the RFC 6902"
        }
    ]
}
```

Reply:
As such, update the `EndpointMetadataGetResponse` please:
```
{
    "namespace": "org.kaaproject.ipc.epmmp.gen.v1",
    "name": "EndpointMetadataGetResponse",
    "type": "record",
    "doc": "Endpoint metadata get response",
    "fields": [
        {
            "name": "correlationId",
            "type": "string",
            "doc": "Message id primarily used to track message processing across services"
        },
        {
            "name": "timestamp",
            "type": "long",
            "doc": "Message creation unix timestamp in milliseconds"
        },
        {
            "name": "timeout",
            "type": "long",
            "default": 0,
            "doc": "Amount of milliseconds since the timestamp until the message expires. Value of 0 is reserved to indicate no expiration."
        },
        {
            "name": "endpointId",
            "type": "string",
            "doc": "Identifier of the endpoint, on behalf of which metadata is requested"
        },
        {
            "name": "eTag",
            "type": [
                "null",
                "string"
            ],
            "default": null,
            "doc": "Identifier for a specific version of a resource"
        },
        {
            "name": "payload",
            "type": [
                "null",
                "bytes"
            ],
            "default": null,
            "doc": "Serialized endpoint metadata content"
        },
        {
            "name": "statusCode",
            "type": "int",
            "doc": "HTTP status code of the request processing"
        },
        {
            "name": "reasonPhrase",
            "type": [
                "null",
                "string"
            ],
            "default": null,
            "doc": "Human-readable status reason phrase"
        }
    ]
}
```